exercises/src/test/scala/fpinscala/testing/exhaustive/PropSpec.scala

```diff
@@ -0,0 +1,178 @@
+package fpinscala.testing.exhaustive
+
+import java.util.concurrent.Executors
+
+import fpinscala.parallelism.Nonblocking.Par
+import fpinscala.parallelism.Nonblocking.Par.ParOps
+import org.specs2.mutable._
+import Prop._
+import Gen._
+import org.specs2.specification.AfterAll
+
+object PropSpec {
+  implicit class PropOps(val p: Prop) extends AnyVal {
+    def quickRun = p.run(100, 100, RNG.Simple(0))
+  }
+}
+class PropSpec extends Specification with AfterAll {
+
+  import PropSpec._
+
+  "forAll" should {
+    "execute property checks with success" in {
+      val props = forAll(choose(10, 20)) { i => i >= 10 && i < 20 }
```

Comment:
could be useful to reuse this across all test

val failedProps1 = ...
val successProps1 = ...
....